### ISSUE TITLE
Accessibility Violation : <ibm-sidenav-menu> The tabbable element's role "group" is not a widget role

### ISSUE THREAD

**Issue by PallaviPattikuntapalli** [IBM/carbon-components-angular#2351]
## Detailed description

IBM Accessibility Checker tool reporting : The tabbable element's role "group" is not a widget role

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug
> Is this issue related to a specific component?

<ibm-sidenav-menu>

> What did you expect to happen? What happened instead? What would you like to see changed?

<ul class="bx--tree-node__children" role="group" tabindex="0" _ngcontent-hyx-c61="">

> What browser are you working in?

Mozila Firefox

> What version of the Carbon Design System are you using?

"carbon-components": "^10.58.1",
"carbon-components-angular": "^4.59.14",

## Steps to reproduce the issue
1.Run the IBM Accessibility Checker tool on component 

![image](https://user-images.githubusercontent.com/108853103/205063353-fcd6e22b-b05d-47b9-b405-9c6b38848568.png)

Element Location :
<ul class="bx--tree-node__children" role="group" tabindex="0" _ngcontent-hyx-c61="">

**Comment by PallaviPattikuntapalli** [comment]
Any updates for this issue.

**Comment by PallaviPattikuntapalli** [comment]
Any updates for this issue.
